Answers.md

```diff
@@ -0,0 +1,15 @@
+Sprint Challenge
+
+1.  Box3 would have the highest specificity as it's last and doubley specific compared to Box.
+
+2.  display:block takes its own space and doesn't wrap into text, display:inline just fits as close as it can.
+
+3.  align-items effects the cross-axis which is opposite the primary one.
+
+4.  Fixed Layout: Does not adjust with the viewport and remains one constant size.
+    Adaptive Layout: Has breakpoints at which content looks good for different specific viewport sizes.
+    Fluid Layout: Adjusts based on % at any breakpoint size.
+    Responsive Layout: A combination of adaptive layout and fluid layout to achieve the best of both worlds.
+
+5.  You use max-width so that on widescreen and ultrawide monitors elements aren't just continually stretched until
+    they look terrible, You set a constant maximum size for the page with solid background around it.
```

Comment:
Exactly! The key word is constraint. Max-width provides a constraint and the the example you provided is just that.